src/modals/MediaDbSearchModal.ts

```diff
@@ -76,6 +76,9 @@ export class MediaDbSearchModal extends Modal {
 
 		const placeholder = 'Search by title';
 		const searchComponent = new TextComponent(contentEl);
+		const toggleComponents = [];
```

Comment:
What do you need this array for? It seems like you are only pushing to it in line 122, but you are never reading from it.